http-server-netty/src/main/java/io/micronaut/http/server/netty/DefaultHttpContentProcessorResolver.java

```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright 2017-2019 original authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.micronaut.http.server.netty;
+
+import io.micronaut.context.BeanLocator;
+import io.micronaut.core.annotation.Internal;
+import io.micronaut.core.io.buffer.ByteBuffer;
+import io.micronaut.core.type.Argument;
+import io.micronaut.core.util.CollectionUtils;
+import io.micronaut.http.MediaType;
+import io.micronaut.http.annotation.Body;
+import io.micronaut.http.server.netty.configuration.NettyHttpServerConfiguration;
+import io.micronaut.web.router.RouteMatch;
+import io.micronaut.web.router.qualifier.ConsumesMediaTypeQualifier;
+
+import javax.annotation.Nonnull;
+import javax.inject.Singleton;
+import java.util.Optional;
+import java.util.Set;
+import java.util.function.Supplier;
+
+
+/**
+ * Default implementation that determines if the body argument (if present) does not expect
+ * decoding of the request body, that the {@link DefaultHttpContentProcessor} should be used,
+ * regardless of the content type of the request.
+ *
+ * If the body argument dictates decoding should occur, the processor that corresponds to
+ * the request content type will be returned.
+ *
+ * @author James Kleeh
+ * @since 1.3.0
+ */
+@Singleton
+@Internal
+class DefaultHttpContentProcessorResolver implements HttpContentProcessorResolver {
+
+    private static final Set<Class> RAW_BODY_TYPES = CollectionUtils.setOf(String.class, byte[].class, ByteBuffer.class);
```

Comment:
should `Object` and `InputStream` be supported as well?

Reply:
I don't really see the benefit of supporting Object and we currently don't support InputStream